rewrite-java/src/main/java/org/openrewrite/java/JavaTemplate.java

```diff
@@ -153,22 +153,20 @@ public Builder imports(String... fullyQualifiedTypeNames) {
 
         public Builder staticImports(String... fullyQualifiedMemberTypeNames) {
             for (String typeName : fullyQualifiedMemberTypeNames) {
-                if (shouldAddImport(typeName)) {
```

Comment:
There is apparently one more reference on line 147, when needs refactoring.

Reply:
that's what I get for using the online editor :)

Reply:
You are very brave! :-)